node/lib/util/submodule_config_util.js

```diff
@@ -71,6 +71,34 @@ const CONFIG_FILE_NAME = "config";
 exports.modulesFileName = ".gitmodules";
 
 /**
+ * Return the relative path from the working directory of a submodule having
+ * the specified `subName` to its .git directory.
+ * @param {String} subName
+ */
+exports.computeRelativeGitDir = function (subName) {
```

Comment:
Oh, I do consult the `.gitmodules` file from the `open` command. This utility is just a little lower -level than that and accepts the URL that was previously read from the `.gitmodules`.